crates/solidity/testing/snapshots/cst_output/HexStringLiteral/invalid_leading_separator/generated/0.4.11.yml

```diff
@@ -6,11 +6,12 @@ Source: >
 Errors: # 1 total
   - >
```

Comment:
I wonder if that also fixes #335?

Reply:
Yes, because we now fail tokens in their entirity